bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/DbLedgerStorage.java

```diff
@@ -712,23 +874,38 @@ public EntryLogger getEntryLogger() {
 
     @Override
     public long getLastAddConfirmed(long ledgerId) throws IOException {
-        throw new UnsupportedOperationException();
+        TransientLedgerInfo ledgerInfo = transientLedgerInfoCache.getIfPresent(ledgerId);
+        Long lac = null != ledgerInfo ? ledgerInfo.getLastAddConfirmed() : null;
+        if (null == lac) {
+            ByteBuf bb = getEntry(ledgerId, BookieProtocol.LAST_ADD_CONFIRMED);
+            bb.skipBytes(2 * Long.BYTES); // skip ledger id and entry id
+            lac = bb.readLong();
```

Comment:
`bb` should be released at this point

Reply:
yes. I was actually fixing that for InterleavedLedgerStorage as well. not sure why it didn't get in this PR. pushing it.